creator-node/test/sync.test.js

```diff
@@ -200,7 +200,7 @@ describe('Test secondarySyncFromPrimary()', async function () {
         if (trackIds[0] === blockchainTrackId) {
           trackOwnerId = userId
         }
-        return ({
+        return {
```

Comment:
is this all. just lint

Reply:
I think so. A lot of the test files aren't formatted like the rest of the codebase so my editor likes to format them on save

Reply:
thats cool - important to confirm tho